client/src/scene/TradeUI.ts

```diff
@@ -0,0 +1,50 @@
+import { SceneGameArena } from "./SceneGameArena";
+import { PlayerColor } from "common/PlayerAttributes";
+import { TradeState } from "common/TradeState";
+
+export class TradeUI {
+    tradeState: TradeState;
+    tradeText: Phaser.GameObjects.Text;
+    tradingUser: number | null;
+    //Eventually should include who was offering the trade
+    tradeMap: { [key in TradeState]: string } = {
+        [TradeState.NoTrade]: "SHIFT to Trade",
+        [TradeState.Offered]: "Trade Requested",
+        [TradeState.Pending]: "SHIFT to Trade with",
+        //likely empty text when accepted
+        [TradeState.Accepted]: "",
+    };
+    constructor(scene: SceneGameArena) {
+        const y = 50;
+        this.tradeState = TradeState.NoTrade;
+        this.tradingUser = null;
+        this.tradeText = scene.add.text(20, y, this.tradeMap[this.tradeState], {
+            fontSize: `20px`,
+            fontFamily: "VT323",
+        });
```

Comment:
Looks like this has yet to be updated to use BitmapText? The trademap will accordingly need to be updated to be lowercase as well.